### PR TITLE
Added Entity.toString() method

### PR DESCRIPTION
**Pull request by csilvestrini**

### CODE REVIEW

File: src/tools/schema2kotlin.ts [hunk 1]
```diff
@@ -148,6 +149,8 @@ class KotlinGenerator implements ClassGenerator {
     this.encode.push(`${fixed}.let { encoder.encode("${field}:${typeChar}", ${fixed}) }`);
 
     this.fieldSerializes.push(`"${field}" to ${fixed}.toReferencable()`);
+
+    this.fieldsForToString.push(`${fixed} = $${fixed}`);
```
Comment (SHeimlich) [inline src/tools/schema2kotlin.ts]: Is this supposed to be a double dollar sign?

Comment (csilvestrini) [reply src/tools/schema2kotlin.ts]: Yep! The output of toString looks like this:

```
override fun toString() = "Foo(bar = $bar, baz = $baz)"
```

Comment (SHeimlich) [reply src/tools/schema2kotlin.ts]: Ahh!

**Pull request merged**